lib/transform.js

```diff
@@ -8,8 +8,9 @@ var through          =  require('through')
 function requireDependencies(src) {
   var deps = findDependencies(src);
   if (!deps.length) return '';
-  return '/* proxyquireify injected requires to make browserify include dependencies in the bundle */;' +
-    requireDeps(deps);
+
+  return '/* proxyquireify injected requires to make browserify include dependencies in the bundle */ /* istanbul ignore next */; (function() { ' +
+    requireDeps(deps) + '});\n\n';
```

Comment:
Please name this function like `makeBrowserifyIncludeModule` or similar, so that people looking at the bundle have an idea what's going on, especially if they see these functions in their heap dumps.